Assets/Text/cqui_text_settings.xml

```diff
@@ -125,6 +125,9 @@
     </Row>
     <Row Tag="LOC_CQUI_GENERAL_SHOWLUXURY" Language="en_US">
       <Text>Show luxuries in top panel</Text>
+    </Row>
+	<Row Tag="LOC_CQUI_GENERAL_SHOWDIPLOBANNER" Language="en_US">
```

Comment:
Indentation is not correct here (and in all `cqui_text_settings_*.xml` files, copy/paste 😉)